joomla/joomla-cms
# [5.0] Update deleted files list in script.php for 5.0.0-alpha3

Pull Request for Issue # .

### Summary of Changes

This pull request (PR) updates the list of files to be deleted on update in file "administrator/components/com_admin/script.php" to recent changes in the 5.0-dev branch in preparation for the upcoming 5.0.0-alpha3 release.

In detail deleted files and folder from following PR's are added:

- PR #40664 files
'/libraries/classmap.php',
'/libraries/extensions.classmap.php',
- PR #41070 following folders and their subfolders and files
'/plugins/editors/codemirror/src/Field',
'/media/vendor/codemirror/theme',
'/media/vendor/codemirror/mode',
'/media/vendor/codemirror/lib',
'/media/vendor/codemirror/keymap',
'/media/vendor/codemirror/addon',

### Testing Instructions

Code review.

Or if you want to make a real test, update a current 4.4-dev nightly build or a 5.0.0-alpha2 to the last 5.0 nightly build to get the actual result, and update a current 4.4-dev nightly build or a 5.0.0-alpha2 to the update package built by Drone for this PR to get the expected result.

### Actual result BEFORE applying this Pull Request

The files and folders mentioned above are still present after updating from a current 4.4-dev nightly build or a 5.0.0-alpha2.

### Expected result AFTER applying this Pull Request

The files mentioned above have been deleted after updating from a current 4.4-dev nightly build or a 5.0.0-alpha2.

### Link to documentations
Please select:
